src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs

```diff
@@ -54,16 +55,17 @@ protected override bool CanFind(IMethodSymbol symbol)
                 predefinedType == actualType;
         }
 
-        protected override Task<ImmutableArray<ReferenceLocation>> FindReferencesInDocumentAsync(
+        protected override Task<ImmutableArray<(SyntaxNode node, ReferenceLocation location)>> FindReferencesInDocumentAsync(
```

Comment:
>(SyntaxNode node, ReferenceLocation location) [](start = 47, length = 45)

When I see this tuple type appear in so many places, I'm thinking we need a named type instead.
Might it make sense to add the `SyntaxNode` into `ReferenceLocation`? 
Otherwise, let's create a struct type to hold the two parts together.

Reply:
We can't add this into ReferenceLocation as ReferenceLocaiton is our language-neutral type for describing references.  But we could make a new named struct to describe this pair.  Note: i would love to use my refactoring.  But it doesn't look available in 15.8...

Reply:
Done! :)